fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -408,6 +403,10 @@ def reduce(*_: Any) -> None:
                 if not self.should_accumulate_grads and self._grad_to_be_reduced[index]:
                     assert param.grad is not None, "Reducing gradients during backward pass, cannot be None"
 
+                    if not self._bucket_flush_callback_set:
+                        Variable._execution_engine.queue_callback(self._flush_buckets)
```

Comment:
I thought the Variable API was deprecated? Is it ok to use this in that case?

Reply:
.. it's pretty useful, that's all I can say :) pytorch and APIs are a wild place

Reply:
Fair enough. Just curious about what the recommendation is for users who were using this. I hope there was a replacement :)

Reply:
looks like it breaks with pytorch 1.5, looking into it..